runtime/tests/interpreter/contract_test.go

```diff
@@ -0,0 +1,156 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package interpreter_test
+
+import (
+	"testing"
+
+	"github.com/onflow/cadence/runtime/interpreter"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInterpretContractUseBeforeInitializationComplete(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("constructor of nested type, always qualified ", func(t *testing.T) {
+
+		t.Parallel()
+
+		_, _ = parseCheckAndInterpretWithOptions(t,
```

Comment:
I guess the return error must be asserted for 'no error'?

Reply:
Good catch! I had added the error return value and then forgot to actually check it for this test case 👍 

Fixed in e02872921db62d2eac8816f7b9aee291f0cd46cc